manuals/manual_convert_schedule_to_events.md

```diff
@@ -0,0 +1,463 @@
+  },
+  // The UNLOCODE of the port which the ship will visit
+  "port": "XXGIL",
+  // Because for our visit the Port Authority has not yet provided us with a portcall identifier we generate our own (for more information see the port call id paragraph below).
+  "portcallId": "PID-CarrierX-077b9b08",
+  "location": {
+    "type": "port",
+    // Our port already has GLNs assigned so we can use those
+    "gln": "006141400"
+  }
+}
+```
+**Note**: *Be aware that the event times should be in UTC (Zulu) time. Since our carrier is currently in the +02:00 time zone, two hours need to be suntracted from the local time. We recommend using a date-time library for this.*
+
+
+### Event types
+| Event name               | Identifier                     | Description                                                                  |
+|--------------------------|--------------------------------|------------------------------------------------------------------------------|
+| ETA Port                 | port.eta.carrier               | The estimated time the ship will arrive at the port.                         |
+| ATA Port                 | port.ata.carrier               | The actual time the ship has arrived in the port.                            |
+| ETA Pilot Boarding Place | pilotBoardingPlace.eta.carrier | The estimated time the ship has arrived at the pilot boarding place          |
+| ATA Pilot Boarding Place | pilotBoardingPlace.ata.carrier | The actual time the ship will arrive at the pilot boarding place             |
+| ETA Berth                | berth.eta.carrier              | The estimated time the ship will arrive at the berth (before it's moored)    |
+| ATA Berth                | berth.ata.carrier              | The actual time the ship has arrived at the berth                            |
+| ETD Berth                | berth.etd.carrier              | The estimated time the ship will depart from the berth (after it's unmoored) |
+| ATD Berth                | berth.atd.carrier              | The actual time the ship has departed from the berth                         |
+| ETD Pilot Boarding Place | pilotBoardingPlace.etd.carrier | The estimated time the ship has depart through the pilot boarding place      |
+| ATD Pilot Boarding Place | pilotBoardingPlace.atd.carrier | The actual time the ship will depart through pilot boarding place            |
+| ETD Port                 | port.etd.carrier               | The estimated time the ship will depart from the port                        |
+| ATD Port                 | port.atd.carrier               | The actual time the ship has departed from the port                          |
+| Cancel berth visit       | berth.cancel.carrier           | A previously planned visit to a specific berth will no longer happen         |
+| Cancel visit             | port.cancel.carrier            | A previously planned visit to the port will no longer happen                 |
+
+### Port Call ID
```

Comment:
This needs clarification and improvement.
Local portcall Id's like Port of Rotterdam's UCRN and other Maritime Single Window identifiers always have preference. Explicitly mention & explain those.

Then, only if the CSS does not contain that, they can instead generate a port call id. Can we have two distinct/clarifying names?

Should we encode the difference in the events? i.e. `portcallID` at top level always is the local portcall id, and we have the internal identifier as a separate field in the context?